src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -98,7 +99,8 @@ export default function DownloadListTable(props: DownloadListTableProps) {
         'success',
         {title}
       )
-      // refetching the statistics will update the download list, so no need to update the file list here.
+      // refetching the statistics will update the download list, but refresh the list immediately since this will take time
+      refetch()
```

Comment:
I noticed that it takes a second for the change to propagate back down to this component.  Let's refresh the list immediately, and we'll end up doing an additional background refetch after the stats change.

Reply:
Could also try invalidating the cache key. Not sure if that will put you into a hard loading state, which is probably worse than waiting for stale data to update. I think what you have is probably the right thing to do.